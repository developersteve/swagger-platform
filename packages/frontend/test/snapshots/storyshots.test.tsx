import { createMount } from '@material-ui/core/test-utils';
import initStoryshots from '@storybook/addon-storyshots';
// Required because Storybook uses require.context but this file is not run through Webpack
import { join } from 'path';

initStoryshots({
  configPath: join(__dirname, '../../.storybook'),
  /* 
    If you remove this, you end up some rather confusing errors.
    See: https://material-ui.com/guides/testing/ and https://github.com/mui-org/material-ui/issues/9243
  */
  renderer: createMount,
});
