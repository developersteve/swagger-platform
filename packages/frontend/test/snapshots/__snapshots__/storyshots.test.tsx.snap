// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots PlanModal Submit 1`] = `[Function]`;

exports[`Storyshots SimpleToolbar Generic 1`] = `[Function]`;

exports[`Storyshots SpecModal Submit 1`] = `[Function]`;
